README.md

```diff
@@ -31,8 +31,10 @@ Whenever you sync with the server, a file with the format `{date in GMT}={X}+{Y}
 {date in GMT}={B}+{Y}.zip
 
 **Resulting transaction file**
-Because `date in GMT of the oldest` < `date in GMT of the youngest`,
-{date in GMT}={X}+{Y}+{Z}.zip
+Resulting transaction file Based on alphabetical order, say if `X < Y`, then: {date in GMT}={X}+{Y}+{Z}.zip 
```

Comment:
Yes, 
`date in GMT of the oldest` < `date in GMT of the youngest` OR
 `date in GMT of the oldest` > `date in GMT of the youngest`
has no effect on resulting file name